<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>item_last</class>
 <widget class="QWidget" name="item_last">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_last_qa,#label_last_praiseoransnum,#label_last_num,#label_last_data{
    color: #868282;
}

#label{
	color: black;
}

#label_last_Qtitle{
	color: black;
}

#textBrowser_last_text{
	border: none;
}</string>
  </property>
  <widget class="QLabel" name="label_last_qa">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_last_praiseoransnum">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_last_num">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_last_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>431</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_last_data">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>141</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_last_Qtitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>22</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
